cmd/gitsign-credential-cache/main.go

```diff
@@ -32,6 +33,7 @@ import (
 var (
 	// Action flags
 	versionFlag = getopt.BoolLong("version", 'v', "print the version number")
+	systemdFlag = getopt.BoolLong("socket-activation", 's', "use systemd socket activation")
```

Comment:
I'd lean towards not assigning a short flag for flags that aren't multi-platform.
```suggestion
	systemdFlag = getopt.Bool("systemd-socket-activation", "use systemd socket activation")
```

Reply:
This getopt lib accepts either short-only, or short-and-long options. long-only are not supported.

Reply:
ack - I'll replace this with cobra in the near term then 😅 (but in another PR, so no further action needed here).